src/Giraffe/EndpointRouting.fs

```diff
@@ -232,39 +270,48 @@ module Routers =
     let TRACE = applyHttpVerbToEndpoints TRACE
     let CONNECT = applyHttpVerbToEndpoints CONNECT
 
-    let route (path: string) (handler: HttpHandler) : Endpoint =
-        SimpleEndpoint(HttpVerb.NotSpecified, path, handler, id)
+    let route (path: string) (aspNetExtensions: AspNetExtension list) (handler: HttpHandler) : Endpoint =
+        SimpleEndpoint(HttpVerb.NotSpecified, path, handler, id, aspNetExtensions)
```

Comment:
Something that has potential 🤔 

This function that currently uses the [id](https://fsharp.github.io/fsharp-core-docs/reference/fsharp-core-operators.html#id) function has this type:

```fsharp
type ConfigureEndpoint = IEndpointConventionBuilder -> IEndpointConventionBuilder
```

I don't know much about it, but maybe we can use it instead of this custom wrapper for Asp.Net extensions https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.builder.iendpointconventionbuilder?view=aspnetcore-9.0